core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/logging/LoggingUtils.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005,2014 WSO2, Inc. http://www.wso2.org
+ * Copyright 2018 WSO2, Inc. http://www.wso2.org
```

Comment:
@menakaj @thanujalk This should have the year that this class created

Reply:
Fixed this in PR https://github.com/wso2/carbon-kernel/pull/1847